test/built-ins/AsyncFunction/is-not-a-global.js

```diff
@@ -8,6 +8,6 @@ description: >
   %AsyncFunction% is not exposed as a global
 ---*/
 
-assert.throws(ReferenceError, function () {
+assert.throws(ReferenceError, function() {
```

Comment:
The thing i find most universally consistent is “function keyword, space, optional name, argument list”, but yo probably don’t want bikeshedding here :-)